server/ntb/macros/valuta.py

```diff
@@ -93,14 +93,24 @@ def get_currency(today):
 def ntb_currency_macro(item, **kwargs):
     # headline
     # this one is the correct one, just that the clock is past 00:00 datetime.datetime.now().date()
-    today_date = datetime.datetime.today() - datetime.timedelta(2)
+    # This date is created just because we want to test monday error
+    today_date = datetime.datetime.today() - datetime.timedelta(3)
 
     # Setting days to go backwards to one, unless it is Monday. Then we go three days back (Friday)
     days = 1
     if today_date.weekday() == 0:
+        # Monday
         days = 3
+    elif today_date.weekday() == 6:
+        # Sunday - just in case someone decides to run this macro on a Sunday
+        today_date = datetime.datetime.today() - datetime.timedelta(2)
+        days = 3
+    elif today_date.weekday() == 5:

```

Comment:
I think this if is not needed because it should show Friday data, that is one day before